web/book/src/SUMMARY.md

```diff
@@ -7,6 +7,8 @@
 - [Syntax](./reference/syntax/README.md)
 
   - [Literals](./reference/syntax/literals.md)
+    - [F-strings](./reference/syntax/f-strings.md)
+    - [S-strings](./reference/syntax/s-strings.md)
```

Comment:
I am +1 on making more of a hierarchy here

...but are these literals? I think technically they're not — they are not constant — PRQL changes them...

WDYT

Reply:
I agree that s-strings and f-strings aren't strictly literals, but the reader will be comforted to see all the "string-like" things lumped together. (I don't think those string-types would be any easier for newcomers to find farther down in the Syntax table of content.) 

Let's commit as-is and see if anyone comments.

Reply:
I'm hesitant to add something that is wrong... What do others think @PRQL/prql-team ?

Is there a way of adding a bit of hierarchy here while keeping it correct?

Reply:
In an [earlier life (wayback machine from March 2023)](https://web.archive.org/web/20230313115935/https://prql-lang.org/book/) we had a "Language Features" section with Strings, F-strings and S-strings (sections 5.8, 5.9, 5.10) grouped together.

Perhaps the Strings paragraphs could be factored from the Literals page into a separate page, with S- and F- string pages (and R-string page?) following, or as a sub-pages?

Reply:
@max-sixty I don't know if others have responded to this PR/suggestion. What do you think?

Reply:
Sorry, was out for a couple of days (2nd child!), just back home :)

> In an [earlier life (wayback machine from March 2023)](https://web.archive.org/web/20230313115935/https://prql-lang.org/book/)

lol

---

I think at the moment two options:
- Change `Literals` to `Literals & Strings`, retain the proposed approach of having F & S strings within thta
- Revert to the existing approach of separating `Literals` from F & S strings

Your call on which (or happy if you want to propose an alternative)

Reply:
OK. I am working on factoring out Strings with sub-headings. Coming soon to a theater near you - probably later today :-)